core/fetcher.go

```diff
@@ -6,11 +6,11 @@ import (
 	"io"
 	"time"
 
+	tmproto "github.com/cometbft/cometbft/proto/tendermint/types"
```

Comment:
Should we start directly importing celestia-core instead of replacing comet? If we anyway on changing imports we can resolve a long standing dependency issue. Core is never going to be in paar with comet with core needed changes upstreamed. We then can simply node usage by removing one more annoying replace, as well as allow having core and comet simultaneously in a dependency graph.

Reply:
I think this is not possible at the moment because node depends on app, which imports two versions of core.

`github.com/cometbft/cometbft => github.com/celestiaorg/celestia-core v1.52.0-tm-v0.38.17`
`github.com/tendermint/tendermint => github.com/celestiaorg/celestia-core v1.45.0-tm-v0.34.35`

This is required in app in order to make the multiplexer work across different versions of comet (34 and 38)

Reply:
I see, thanks for explaining. Hopefully after v5 can resolve this